packages/connect-explorer-webextension/README.md

```diff
@@ -0,0 +1,25 @@
+# @trezor/connect-explorer-webextension"
+
+This package contains Trezor Connect Explorer webextension.
+
+## Dev
+
+-   Build it with other Trezor Connect source, that will give you the URL that is going to be used for Trezor Popup:
+
+```
+yarn build:libs && \
+__TREZOR_CONNECT_SRC=http://localhost:8088/ yarn workspace @trezor/connect-explorer-webextension build && \
+yarn workspace @trezor/connect-explorer dev
+```
+
+This provides localhost version of popup and connect with connect explorer web extension. It is possible to run it with legacy connect src instead providing different `__TREZOR_CONNECT_SRC=https://suite.corp.sldev.cz/connect/develop/`.
+
+-   Build it with non-default npm Trezor Connect source:
+
+Env variable `__NPM_TREZOR_CONNECT_SRC__` can be use to build it with difference Trezor Connect NPM source `trezor-connect.js`:
```

Comment:
I don't see `__NPM_TREZOR_CONNECT_SRC__`implemented anywhere. I guess it should affect 
```
const CONNECT_WEB_PATH = path.join(__dirname, '..', '..', 'connect-web');
```
in webpack config. maybe it could be used in combination with? https://www.npmjs.com/package/fetch-write-webpack-plugin

Reply:
Yes, this part of the README is outdated from the initial way I had for building it with the scripts. Now it is all with webpack so it has to be implemented.